### DISCUSSION TITLE
Webhook ReleaseEvent : prerelease -> release notification

### DISCUSSION THREAD

**Discussion by leocourson** [community/community#24439] [Apps, API and Webhooks]
<p>Hello everyone !</p>
<p>I’m currently developping a changelog application that used the webhook : <strong>releaseEvent</strong> to recover data.</p>
<p>The 18 April of 2019 new <strong>releaseEvent actions</strong>  where added (<a href="https://developer.github.com/changes/" rel="nofollow noopener">https://developer.github.com/changes/</a>) :  <strong>created, </strong>** edited,  <strong><strong>deleted, </strong></strong> prereleased,  **<strong>unpublished</strong></p>
<p>I just want to receive a webhook when <strong>a prerelease change to release</strong> (prerelease -&gt; release) but :</p>
<ul>
<li>The <strong>edited</strong>  action is triggered only on name and body modification ( <strong>not the prerelease parameter</strong> )</li>
<li>The <strong>prereleased</strong> action is triggered only when a release change to prerelease <strong>BUT NOT THE REVERSE</strong>
</li>
</ul>
<p><strong>Is there something I’m missing ?</strong></p>
<p><strong>What is the point of the prereleased action ?</strong> (release -&gt; prerelease shouldn’t happen that much)</p>
<p><strong>UPDATE (24/04/2019)</strong></p>
<p><strong>Response from GitHub Support : </strong></p>
<p>Thanks for reaching out, Leo. Please find answers and notes inline below.</p>
<ul>
<li>Is there something I’m missing ?</li>
</ul>
<p>I don’t think you’re missing anything. I agree it would be useful if there was a way to get notified when a prerelease is made into a full release, so I passed your feedback along to the team to consider for the future.</p>
<ul>
<li>What is the point of the prereleased action ? (release -&gt; prerelease shouldn’t happen that much)</li>
</ul>
<p>The point is to be notified when release is made into a prerelease. I understand that this doesn’t seem useful to you, but different users have different use-cases and those other users find it useful. How often something happens doesn’t really make a webhook more or less useful. Even if it happens rarely - you want to be notified when it happens.</p>
<p>Thanks again for taking the time to share your feedback.</p>
<p><strong>UPDATE (10/09/2019)</strong></p>
<p>A new action " <strong>released</strong>" have been added for the webhook <strong>releaseEvent</strong> and is sent when a prerelease change to a release.</p>
<p>Thank to the development team :tada:</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hey <a class="mention" href="/leocourson">@leocourson</a>, Thanks a lot for coming back and sharing support’s info! We’re looking forward to seeing you around!</p>
